addons/point_of_sale/static/src/app/navbar/proxy_status/proxy_status.js

```diff
@@ -24,7 +24,7 @@ export class ProxyStatus extends Component {
                 iface_print_via_proxy,
                 iface_cashdrawer,
                 iface_electronic_scale,
-            } = this.pos.config;
+            } = this.pos["pos.config"];
```

Comment:
I really prefer having `pos.config, pos.company, pos.currency` for singleton records, instead of indexing them with the model name.

Reply:
I haven't seen the whole picture but It feels weird to need to access `["pos.config"]` from `pos`.

Reply:
> I'm not really sure about that fact, all models are now accessible by their name, do we really need an exception for these one ?

My way of thinking is that `pos_store` object should just be another relational model, such that it will have fields like `config_id`, `currency_id`, `current_session_id`, etc. These fields will be access same way as other fields of the models loaded in the frontend.